public/content/lessons/2016/cross-products-extended/index.mdx

```diff
@@ -1,10 +1,256 @@
+
+<Figure
+    image="./figures/the-idea/DeterminantOfThreeVectorsVariableU.svg" 
+    video="./figures/the-idea/DeterminantOfThreeVectorsVariableU.mp4" 
+/>
+
+So here we are, out in the mathematical wild with a linear transformation that outputs numbers. The idea of duality should be perking up in your mind. What we're going to do is find the matrix that describes this transformation, which will be a 1x3 matrix since the transformation goes from three dimensions to one. In other words, we're looking for a 1x3 matrix such that multiplying this matrix by some vector $\vec{\mathbf{u}}$ gives the same result as plugging in $\vec{\mathbf{u}}$ to the first column of a 3x3 matrix whose other two columns have the coordinates of $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, and computing the determinant.
+
+<Figure
+    image="./figures/the-idea/1By3MatrixEncodingThe3dTo1dTransformation.svg" 
+/>
+
+If you think in terms of duality and turn this matrix on its side, we're looking for a special 3d vector, that I'll call $\vec{\mathbf{p}}$, such that taking the dot product between $\vec{\mathbf{p}}$ and any other vector $\vec{\mathbf{u}}$ gives the same result as plugging in $\vec{\mathbf{u}}$ to the first column of a matrix and computing the determinant.
+
+<Figure
+    image="./figures/the-idea/3dTo1dTransformationAsVector.svg" 
+/>
+
+I'll get to the geometry of this in just a moment, but right now let's dig in and think about what this means computationally. Taking the dot product between $\vec{\mathbf{p}}$ and $\vec{\mathbf{u}}$ will give 
+
+$$
+(\text{something}) \cdot x + (\text{something}) \cdot y + (\text{something}) \cdot z
+$$
+
+Those somethings are the coordinates of $\vec{\mathbf{p}}$. When you compute the determinant on the right, you can organize it to look like 
+
+$$
+(v_2 w_3 - v_3 w_2) \cdot x + (v_3 w_1 - v_1 w_3) \cdot y + (v_1 w_2 - v_2 w_1) \cdot z
+$$
+
+This shows what those "somethings" are, and hence what the coordinates of the vector $\mathbf{p}$ are.
+
+<Figure
+    image="./figures/the-idea/DotProductBetweenPAndU.svg" 
+/>
+
+So the answers to what those "somethings" are will give the coordinates of the vector $\vec{\mathbf{p}}$ that we're looking for.
+
+<Figure
+    image="./figures/the-idea/DotProductBetweenPAndUComponentsOfP.svg" 
+/>
+
+But this should feel very familiar to anyone who's actually worked through a cross product computation before: Collecting the constant terms that are multiplied by $x$, $y$ and $z$ like this is no different from plugging in the symbols $\hat{\imath}$, $\hat{\jmath}$ and $\hat{k}$, and seeing which coefficients aggregate on each of these terms. It's just that plugging in $\hat{\imath}$, $\hat{\jmath}$ and $\hat{k}$ is a way of signaling that we should interpret these three coefficients as coordinates of a vector.
```

Comment:
Because of the line "But this should feel very familiar to anyone who's actually worked through a cross product computation before" I think it makes sense to add some plug-and-chug type practice problems to the regular cross products lesson. Is the best place for those questions at the very end in their own section?

Reply:
No strong preference, but we could throw one such exercise at the end of the first Cross Product article.